app/models/conversion/involuntary/task_list.rb

```diff
@@ -35,7 +35,8 @@ class Conversion::Involuntary::TaskList < TaskList::Base
         Conversion::Involuntary::Tasks::SingleWorksheet,
         Conversion::Involuntary::Tasks::SchoolCompleted,
         Conversion::Involuntary::Tasks::ConditionsMet,
-        Conversion::Involuntary::Tasks::ShareInformation
+        Conversion::Involuntary::Tasks::ShareInformation,
+        Conversion::Involuntary::Tasks::TellRegionalDeliveryOfficer
```

Comment:
This is the start of the 'After opening' section.

Reply:
I added the section string & everything 🤦